### ISSUE TITLE
`HEAD` verb for table api

### ISSUE THREAD

**Issue by calpaterson** [calpaterson/csvbase#71]
# Brief overview

AS AN API client

I WANT a way just to get the metadata about a table

SO THAT I can return information about last updated/created/etc

# Additional details

**Comment by equirke** [comment]
Seems [done](https://github.com/calpaterson/csvbase/blob/6e34f597f09b7d23d88389a767199c8dc43f35f8/csvbase/web/main/bp.py#L214)!

**Comment by calpaterson** [comment]
> Seems [done](https://github.com/calpaterson/csvbase/blob/6e34f597f09b7d23d88389a767199c8dc43f35f8/csvbase/web/main/bp.py#L214)!

Yes, that's right, I think something was causing it to get hit frequently on the prod site so I ha to put in the optimisation :)
